Module/Resolve Error with React 17

**Describe the bug**
Trying to use the rich textarea component fails to compile using Node 16 or 17 and React 17.

```
ERROR in ./node_modules/rich-textarea/lib/index.mjs 1:0-56

Module not found: Error: Can't resolve 'react/jsx-runtime' in '<project>/node_modules/rich-textarea/lib'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```
**To Reproduce**
Import and use the component.

**Expected behavior**
The project should compile.

**Platform:**
 - OS: macOS Monterey 12.3.1
 - Package Version: ^0.14.0
 - React Version: ^17.0.2
 - Node Version: 16.13.0, 17.9.0 (both tested)

--- a/package.json
+++ b/package.json
@@ -33,6 +33,7 @@
     "@types/jest": "27.4.0",
     "@types/node-emoji": "1.8.1",
     "@types/react-highlight-words": "^0.16.4",
+    "babel-plugin-replace-import-extensions": "1.0.1",
     "jest": "27.5.1",
     "kuromojin": "3.0.0",
     "monaco-diff": "1.1.1",

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -23,7 +23,13 @@ export default {
       exclude: ["src/**/*.spec.*"],
     }),
     getBabelOutputPlugin({
-      plugins: ["@babel/plugin-transform-react-pure-annotations"],
+      plugins: [
+        "@babel/plugin-transform-react-pure-annotations",
+        [
+          "replace-import-extensions",
+          { "^react/jsx-runtime$": "react/jsx-runtime.js" },
+        ],
+      ],
     }),
     terser({
       compress: { passes: 3 },

